libraries/abstractions/pkcs11/CMakeLists.txt

```diff
@@ -100,6 +100,34 @@ else()
     )
 endif()
 
+afr_module(NAME pkcs11_psa INTERFACE)
```

Comment:
Could you wrap this module in an else if block following the pattern above, using `PSA_IN_USE` for example as a variable, so this is not included in all the platforms

Reply:
Done.